### PR TITLE
use asyncio.run directly

### PR DESCRIPTION
**Pull request by fishbone**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/workflow/workflow_storage.py [hunk 1]
```diff
@@ -57,8 +57,7 @@
 
 # TODO: Get rid of this and use asyncio.run instead once we don't support py36
```
Comment (pcmoritz) [inline python/ray/workflow/workflow_storage.py:58]: This comment should be removed too :)

Comment (pcmoritz) [reply python/ray/workflow/workflow_storage.py:58]: (just randomly saw this, you probably planned to do that anyways)

Comment (iycheng) [reply python/ray/workflow/workflow_storage.py:58]: Hi @pcmoritz  this is just a PR to show that simply using `asyncio.run` won't work. I think I'll close this.

**Comment by iycheng** [comment]
Close for demo purpose

**Pull request closed without merging**